apps/server/src/core/error/utils/error-response.utils.ts

```diff
@@ -0,0 +1,65 @@
+import { HttpException, InternalServerErrorException } from '@nestjs/common';
+import { ApiValidationError, BusinessError } from '@shared/common';
+import _ from 'lodash';
+import { ApiValidationErrorResponse, ErrorResponse } from '../dto';
+import { FeathersError } from '../interface';
+import { ErrorUtils } from './error.utils';
+
+export class ErrorResponseUtils {
+	static createErrorResponse(error: unknown): ErrorResponse {
+		let response: ErrorResponse;
+
+		if (error instanceof Error) {
```

Comment:
Diese Prüfung würde ich in die einzelnen Guards verlagern. Dann hast du hier keine verschachtelten ifs.